src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_NullablePublicOnly.cs

```diff
@@ -0,0 +1,199 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public class AttributeTests_NullablePublicOnly : CSharpTestBase
+    {
+        [Fact]
+        public void ExplicitAttribute_FromSource()
+        {
```

Comment:
>{ [](start = 8, length = 1)

`[NullablePublicOnly]` should be disallowed in source. Do we have a test for that?

Reply:
https://github.com/dotnet/roslyn/issues/36457